internal/telemetry/event.go

```diff
@@ -332,6 +332,12 @@ func withUserAgent() EventOpt {
 	}
 }
 
+func WithIndexType(indexType string) EventOpt {
```

Comment:
since there are other type of indexes 
```suggestion
func WithSearchIndexType(indexType string) EventOpt {
```